lua/nvim-treesitter/install.lua

```diff
@@ -584,26 +577,23 @@ M.install = a.sync(function(languages, options)
     a.main()
     log.info('Installed %d/%d parsers', done, #tasks)
   end
-end, 2)
+end
+
+M.install = a.sync(install, 2)
```

Comment:
```suggestion
M.install = a.sync(install, 3)
```
(Or is the `_callback` option special?)

Reply:
I was assuming I was missing something here. Thanks for explaining.